core/stdlib/std.ncl

```diff
@@ -2148,6 +2148,28 @@
       "%%
       = fun field r => r."%{field}",
 
+    getor
+      : forall a. String -> a -> { _ : a } -> a
+      | doc m%%"
+        Returns the field of a record with given name or default value,
+        if there is no such field.
+
+        # Examples
+
+        ```nickel
+        std.record.get "tree" 3 { one = 1, two = 2 } =>
+          3
+
+        std.record.get "one" 11 { one = 1, two = 2 } =>
+          1
+        ```
+      "%%
+      = fun field default_value r =>
+        if std.record.has_field field r then
+          std.record.get field r
```

Comment:
Same thing here - this actually the definition of `get`.

```suggestion
          field."%{r}"
```

Reply:
Sorry, I confused the record and the field @olorin37 - it's actually `r."%{field}"`. This is why the test are failing (the stdlib doesn't typecheck this way, our daily reminder that static typing is valuable :slightly_smiling_face: )